File: Programmers/py/lev0/영어가 싫어요/solution.py
def solution(numbers):
    alpha=["zero","one","two","three","four","five","six","seven","eight","nine"]
    for idx,val in enumerate(alpha):
        numbers = numbers.replace(val, str(idx))
    return int(numbers)


ans = "onetwothreefouronefivesixseveneightnine"	
print(solution(ans))

File: Programmers/java/lev0/영어가 싫어요/solution.java
class Solution {
    public long solution(String numbers) {
        String [] alpha= new String[] {"zero","one","two","three","four","five","six","seven","eight","nine"};

        for (int i=0;i<alpha.length;i++)
            numbers = numbers.replace(alpha[i],String.valueOf(i)); 
        return Long.parseLong(numbers);
    }
}